bin/extract_to_arb.dart

```diff
@@ -86,30 +104,30 @@ void main(List<String> args) {
     print(parser.usage);
     exit(0);
   }
-  var allMessages = {};
+  Map<String, dynamic> allMessages = {};
```

Comment:
If this is helping you with migrating then I have no concerns. But generally - for local variables - we prefer to use `var`, and put any necessary type args on the right as necessary to help type inference. So:

```
var allMessages = <String, dynamic>{};
```

Reply:
We do specify types outside a method however, like for fields or top-level variables.

Reply:
Ah, that's interesting. I personally don't like using `var` a lot, but I can probably adjust to team standards :) I dislike having to chase down call trees to see which type a variable has, especially if I do not have an IDE to help me, for example looking at code here on github.

Reply:
Fair warning, there are still people on both sides of this on the Dart team (and the Flutter SDK has its own code style conventions). Using `var` is what we recommend as our official coding standards however, and what has been converged on in google3 as well.

The relevant bit of the code style guide: https://dart.dev/guides/language/effective-dart/design#dont-redundantly-type-annotate-initialized-local-variables

It's more terse and compact (and modern?). It does leave you guessing in some circumstances about what the actual type might be.

Reply:
I myself just adopt to whatever the code style is of the surrounding code or repo.